app/views/layouts/application.html.erb

```diff
@@ -10,17 +10,20 @@
   </head>
     <nav class="nav">
       <% if current_user %>
-      <a class="nav-link" href="#">Home</a>
-      <a class="nav-link" href="#">Friends</a>
-      <a class="nav-link active" href="#">Carbon Calculator</a>
-      <a class="nav-link" href="#">Offsets</a>
-      <a class="nav-link" href="#">Resources</a>
-      <a class="nav-link" href="#">Log Out</a>
-    <% else !current_user %>
-      <a class="nav-link" href="#">Log In</a>
-    <% end %>
+        <a class="nav-link" href="/dashboard">Home</a>
+        <a class="nav-link" href="/friends">Friends</a>
+        <a class="nav-link active" href="/carbon_calc">Carbon Calculator</a>
+        <a class="nav-link" href="/offsets">Offsets</a>
+        <a class="nav-link" href="/resources">Resources</a>
+        <a class="nav-link" href="/log_out">Log Out</a>
```

Comment:
Genuine question: should this be the same syntax as the other links we have throughout the project? 
ie: ` <%= link_to 'Login with Google', "/auth/google_oauth2" %> `

Reply:
Same syntax worked on my end.